Casks/flutter.rb

```diff
@@ -1,6 +1,6 @@
 cask 'flutter' do
-  version "1.5.4-hotfix.2"
-  sha256 "0b4cacbe84fd68cc9cfe213a90ddcdba558e45c19be30c6173c07a318be34014"
+  version "v1.7.8+hotfix.3"
```

Comment:
Could you please remove the `v` prefix from the version? It's already included in the URL and is not needed here.